internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,54 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	if data == "" {
+		return 0, 0, errors.New("empty input")
+	}
+
+	arrData := strings.Split(data, ",")
+	if len(arrData) != 2 {
+		return 0, 0, errors.New("wrong input")
+	}
+
+	steps, err := strconv.Atoi(arrData[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Ошибки лучше оборачивать
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```